### PR TITLE
Datetime to date

### LINKED ISSUE #64: Change datatype for any dates to Date
Currently stored as DateTime

### PR DESCRIPTION
**Pull request by LHJE**
### Code Highlights:
 * Changes all `datetimes` and `DateTimes` to `date` or `Date`, excepting the `created_at` and `updated_at` in the schema.
### Where should the reviewer start?
 * At the `schema.rb`
### Have Tests Been Added?
- [x] No.
- [ ] Yes, but all tests are not passing.
- [ ] Yes, and all are passing.
### Issues:
* Addresses #64 
* Closes #64 
### Tracking Consistency:
- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [x] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] looked at PR preview to check spelling, syntax, formatting, and completion
- [x] Rubocop Violations: the same `get_user` error

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -46,7 +46,7 @@
   create_table "users", force: :cascade do |t|
     t.string "name"
     t.string "email"
-    t.datetime "birthdate"
+    t.date "birthdate"
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
   end
```
Comment (LHJE) [inline db/schema.rb:52]: This is where the major changes are really made

**Review by LHJE — COMMENTED** [review]
Pretty simple switch over!

**Review by leahriffell — APPROVED** [review]

**Comment by leahriffell** [comment]
Thanks, Luke! Should we also change the Eras and Events factories to generate a random Date?

**Review by AngelaGuardia — APPROVED** [review]
Looks good Luke. With regards to @leahriffell 's question about the factories, I updated those in my PRs. The changes should be reflected once those are merged!

**Review by dhalverson — APPROVED** [review]
Reviewed, will have to double check the entire file once everything is merged

**Pull request merged**